src/text_file_functions.cpp

```diff
@@ -896,6 +896,19 @@ QString AOApplication::get_custom_realization(QString p_char)
   return get_sfx_suffix(get_sounds_path(f_result));
 }
 
+bool AOApplication::get_pos_is_judge(QString p_pos)
+{
+  QStringList positions = read_design_ini("judges", get_background_path("design.ini")).split(",");
+  if (positions.size() == 1 && positions[0] == "")
```

Comment:
why not <= 1

Reply:
nvm I'm dum

Reply:
Why would `positions` not be empty if `judges` is undefined in the INI file? Now that's an omni-ism.